integrations/octopus/.port/resources/port-app-config.yml

```diff
@@ -0,0 +1,96 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: project
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          identifier: .SpaceId

```

Comment:
use the project .Id instead of spaceId

Reply:
All IDs that can form a relationship have been updated to and removed from properties